includes/orders/functions/refunds.php

```diff
@@ -36,7 +36,7 @@ function edd_is_order_refundable( $order_id = 0 ) {
 	}
 
 	// Only completed orders can be refunded.
-	if ( 'complete' !== $order->status ) {
+	if ( ! in_array( $order->status, array( 'complete', 'publish', 'partially_refunded', true ) ) ) {
```

Comment:
What's the `true` for? Or was that meant to be outside the `array()` ?

Reply:
It tells `in_array()` to a strict type check recommended by WPCS: https://github.com/WordPress/WordPress-Coding-Standards/blob/d2ef2b5370dca0a11c7735bd9e14d711f8e18f32/WordPress/Sniffs/PHP/StrictInArraySniff.php

Reply:
Edit: Oh, yes. ☕️